README.adoc

```diff
@@ -244,8 +238,19 @@ $ bundle exec rake features BOX=cdk PROVIDER=libvirt
 
 # Run against ADB and CDK (boxes are comma separated)
 $ bundle exec rake features BOX=cdk,adb
+
+# Run against ADB and CDK using VirtualBox and Libvirt
+$ bundle exec rake features BOX=cdk,adb PROVIDER=libvirt,virtualbox
 -----------------------------------------------------
 
+The _features_ task will transparently download the required Vagrant
+boxes and cache them in _.boxes_. The cache can be cleared
```

Comment:
no, you want to be outside the _build_ directory, since this gets deleted and then you loose the cached boxes